tools/ci/tc/tests/test_valid.py

```diff
@@ -63,7 +63,7 @@ def test_verify_payload():
         with open(data_path(filename)) as f:
             event = json.load(f)
 
-        with mock.patch("tools.ci.tc.decision.get_fetch_rev", return_value=(event["after"], None)):
+        with mock.patch("tools.ci.tc.decision.get_fetch_rev", return_value=(event["after"], None, None)):
```

Comment:
Looks like this predates this PR, but that return_value doesnt seem right - `event["after"]` is the first return? (What about event["ref"]?)